substrate/frame/election-provider-multi-block/src/template.hbs

```diff
@@ -0,0 +1,88 @@
+// This file is part of Substrate.
+
+// Copyright (C) Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+{{header}}
+//! Autogenerated weights for `{{pallet}}`
+//!
+//! (THIS IS A SPECIAL TEMPLATE FOR `election-provider-multi-block` PALLET)
```

Comment:
Can you add one comment here why we need to copy it instead of using the default?  
I assume that we need to tell the bench bot about it.

Reply:
we cannot benchmark this pallet in CI yet because it requires a large genesis config to be set in the runtime, and I didn't want to deal with configuring that yet. The large genesis config would slow down all benchmarks if created for all, and I don't know the exact syntax to make it an option just for this pallet in the bench CI. 

Atm the only reason to have this separate template is because I don't want the `trait WeightIfo`.